### PR TITLE
Conda edits for JOSE paper

### LINKED ISSUE #472: Consider adding --from-history option to "conda list --export"
I suggest that you consider including the --from-history argument when advising to export the conda list.  Although it will not reproduce the exact environment I have found that the argument makes for much easier to understand yml files.  Just a recommendation, I realize you also don't want to overburden the learner with too many details.

BTW. Thank you for this great tutorial.  I happen to be working on a similar one for python and having this as a reference will save me a lot of time!  I learned a lot.

### LINKED ISSUE #473: Consider adding some narrtion to the conda videos
Although, I realize that all of the key information is in the accessible text of the tutorial and the video is only there as additional reference, I still recommend adding verbal content to the videos and allow YouTube to generate CC transcripts. I found the video confusing without the text and it took me a second to find the text describing what was in the video.  I realize this is a minor point but if the video just had someone narrating what was going on I think it could add a lot.  Overall great tutorial, --thank you

### PR DESCRIPTION
**Pull request by marisalim**
## PR Checklist

- [ ] Release label added - when is the next release?
- [x] PR category label added
- [x] PR mergeable
- [x] Clean build logs - nearly, broken github action for the reviewer checklist?
- [x] Linked related issues
- [ ] Colorblindness test https://www.toptal.com/designers/colorfilter/ - n/a

## PR Description
per suggestions from Dirk's JOSE review #472 , #473 
- added --from-history export as a note in the Method 3 yaml section
- added some more context for the vidlets

**Preview link**
https://cfde-training-and-engagement--475.com.readthedocs.build/en/475/General-Tools/Introduction-to-Conda/conda2/

## Review format
comments or direct edits

## Timeline
aiming for before Thanksgiving, or sooner 🦃

### CODE REVIEW

**Review by s-canchi — COMMENTED** [review]
For the videos, would it also make sense to list few big steps as bullet points ? The goal of the video is summarized in a sentence but it can be challenging to follow through without additional written text or narration. Adding CC files for specific timestamps would also be awesome but would vary in format from rest of the website.

File: docs/General-Tools/Introduction-to-Conda/conda2.md [hunk 1]
```diff
@@ -49,13 +47,15 @@ The binder auto-generates a very long command prompt. We will shorten it to `$`:
...
 
 We are currently in the `(base)` conda environment.
```
Comment (s-canchi) [inline docs/General-Tools/Introduction-to-Conda/conda2.md]: So the `(base)` doesn't explicitly show up on the prompt till later in the lesson when they exit out of their first environment. Should we add a note about that ?

**Comment by marisalim** [comment]
> 
> 
> For the videos, would it also make sense to list few big steps as bullet points ? The goal of the video is summarized in a sentence but it can be challenging to follow through without additional written text or narration. Adding CC files for specific timestamps would also be awesome but would vary in format from rest of the website.

all the code steps for the videos are right next to the vidlets (either directly above or below).

File: docs/General-Tools/Introduction-to-Conda/conda2.md (hunk 1 above)
Comment (marisalim) [reply docs/General-Tools/Introduction-to-Conda/conda2.md]: sure!

**Review by s-canchi — APPROVED** [review]
lgtm!

**Pull request merged**